scenarioo-server/build.gradle

```diff
@@ -56,7 +57,7 @@ generateTypeScript {
             'org.scenarioo.rest.search.SearchEngineStatus',
             'org.scenarioo.model.docu.aggregates.usecases.UseCaseSummary',
             'org.scenarioo.model.docu.aggregates.branches.BranchBuilds',
-            'org.scenarioo.rest.sketcher.issue.dto.SketchIds',
+            'org.scenarioo.rest.sketcher.issue.dto.SketchIds'
```

Comment:
why did you remove the comma? Now it shows a change where none is

Reply:
Sorry, was adding the new DTO at the end of the list first and must have overlooked that there was already a comma before.